src/api/app/assets/stylesheets/webui/responsive_ux/_keys_and_certs.scss

```diff
@@ -0,0 +1,92 @@
+.key-and-certificates-card {
```

Comment:
Hm, I don't understand. What is the use to just "rename" the bootstrap classes?

Reply:
In this case its not super-clear what's the benefit of doing it so, but if you look at [this other element](https://github.com/openSUSE/open-build-service/pull/9156/files#diff-e97de64bb90a9262713f71a9c2538b84R17), placing the bootstrap classes inline would render this:
```
.row
  .col-12.col-md-6.mb-2.p-3.p-md-3
    .card
```
Which is uglier...
And there's also the benefit of using some classes with meaningful semantics

Reply:
Anyway, @saraycp also showed concerns regarding this use of `@extend` so I will fall back to my previous changes that were inlined

Reply:
> Which is uglier...

Uglier how? If I know bootstrap, I perfectly understand what is going on there. And hiding this in some CSS file removes my understanding of what is going on.

Reply:
Its uglier in the sense that it has no semantic, this line could be anywhere else and no one cared.

About hiding the classes in some CSS I can only say that I used the same logic I use to abstract some code behind a function or a class. So `.key-and-certificates-card` is useful to me in telling me what this is right away, but `.col-12.col-md-6.mb-2.p-3.p-md-3` at first glance is just gibberish... Don't you think?

As I said earlier, I'm falling back to inline the bootstrap classes for consistency's sake.